gh-actions/config/karma/test.karma.conf.js

```diff
@@ -0,0 +1,24 @@
+const shared = require('../../../shared/karma/shared.karma.conf');
+
+/**
+ * GitHub Actions platform overrides.
+ * @name getConfig
+ * @param {Object} config
+ */
+function getConfig(config) {
+  config.set({
```

Comment:
Created a new `gh-actions` subdirectory to keep the Travis stuff untouched. 

I also overwrote the timeout configs because builds would take forever to fail with what we've set in the shared config. I'd rather the builds fail fast than for us to wait 2+ hours and restart it. (We can discuss whether or not these changes should be applied to the shared config or not.)

Reply:
Perhaps your builds are more localized and this change won't be a problem, but those limits were increased on purpose.